src/test/java/nextstep/subway/line/LineTestHelper.java

```diff
@@ -0,0 +1,57 @@
+package nextstep.subway.line;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import nextstep.subway.dto.LineRequest;
+import nextstep.subway.dto.LineUpdateRequest;
+import org.springframework.http.MediaType;
+
+public class LineTestHelper {
+    private LineTestHelper() {
+    }
+
+    static Long 지하철_노선_생성_하고_ID_응답(LineRequest lineRequest) {
+        return 지하철_노선_생성(lineRequest)
+                .jsonPath().getLong("id");
+    }
+
+    static ExtractableResponse<Response> 지하철_노선_생성(LineRequest lineRequest) {
+        return RestAssured.given().log().all()
+                .body(lineRequest)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().post("/lines")
+                .then().log().all()
+                .extract();
+    }
+
+    static ExtractableResponse<Response> 지하철_노선_ID로_조회(Long lineId) {
```

Comment:
```suggestion
    static ExtractableResponse<Response> 지하철_노선_조회(Long lineId) {
```

`_ID로`와 같은 구현에 대한 내용은 제거해도 괜찮을 것 같은데 어떻게 생각하시나요?

Reply:
아하 기존 코드라면 이런 부분들이 오히려 제거되는게 이점이 많다고 같이 공감합니다.
하지만 테스트케이스를 위한 코드에 한글명으로 작성한 메소드라 이왕 적는거 메소드명을 최대한 친절하게 적는 것이 좋을 것이라 판단했습니다~

Reply:
다른 조건으로 조회할 경우 오버로딩을 통해 표현할 것이라 예상했는데요!
id로 조회한다는 내용을 메서드명에 작성한다면 어떠한 장점이 있을까요?

Reply:
아 네 위에 말씀드렸듯,, 테스트 케이스의 코드라서 메소드명만으로 한번에 확인이 가능할 것 같다는 생각으로 작성한거였습니다~
(물론 parameter 필드의 네이밍을 보고 알 수도 있겠지만요)